corehq/apps/users/models.py

```diff
@@ -2402,6 +2402,14 @@ def get_admins_by_domain(cls, domain):
             if web_user.is_domain_admin(domain):
                 yield web_user
 
+    @classmethod
+    def get_billing_admins_by_domain(cls, domain):
+        from corehq.apps.users.role_utils import UserRolePresets
```

Comment:
If this import doesn't cause a circular import, I'd move it outside of this function to the top of the module. I don't know that we have standards around this, but my impression was that we're only using local imports to avoid circular imports because they're a code smell.

Reply:
yep this was because of circular import issue